codegenerator/cli/src/config_parsing/entity_parsing.rs

```diff
@@ -851,15 +995,20 @@ pub enum UserDefinedFieldType {
 }
 
 impl UserDefinedFieldType {
-    fn from_obj_field_type(obj_field_type: &ObjType<'_, String>) -> Self {
+    fn from_obj_field_type(
+        obj_field_type: &ObjType<'_, String>,
+        pg_type_modifications: &PgTypeModifications,
```

Comment:
This second argument is used to tell it if there are additional customizations on the Postgres type.